setup.py

```diff
@@ -22,10 +22,10 @@
     package_dir={"twisted": "daphne/twisted"},
     packages=find_packages() + ["twisted.plugins"],
     include_package_data=True,
-    install_requires=["twisted[tls]>=18.7", "autobahn>=0.18", "asgiref~=3.2"],
+    install_requires=["twisted[tls]>=20.3", "autobahn>=20.4", "asgiref~=3.2"],
```

Comment:
> major version bump

If you do so, I propose to also drop support for Python 3.5 (at least formally) as it will reach its end of life in 2 months, see https://devguide.python.org/#status-of-python-branches.